Adding initial draft of `ww-proseq` WILDS WDL pipeline (#337)
getwilds/wilds-wdl-library: modules/ww-bowtie2/README.md

### `bowtie2_align`

> Aligns sequence reads to a reference genome using Bowtie 2. Supports named sensitivity presets, post-alignment MAPQ + SAM-flag filtering, capture of unaligned reads (e.g. for rRNA depletion), and an extra-args escape hatch for tool-specific flags.

Reviewer: ```suggestion
Aligns reads to a reference using Bowtie 2, optionally filtering the output using Samtools
```
As with the WDL, I would keep this brief and let the parameter descriptions speak for themselves.
